src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/AssemblyHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reflection;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon
+{
+    public static class AssemblyHelper
+    {
+        public static string GetAssemblyArtifactBinPath(
```

Comment:
@kkeirstead, I'm add this new method that helps in calculating the entry point assembly path based on the current test assembly location. It's basically the same algorithm the rest of the tests had previously used, but consolidated and added an additional means of choosing which TFM (target framework moniker e.g. `netcoreapp3.1`, `net5.0`, or `net6.0`) folder to find the assembly. You could probably use this in your new test that you are working on. I'll try to get this merged in soon and merged to the feature/triggers branch.